setup.py

```diff
@@ -56,6 +42,31 @@ def run(self):
         finally:
             os.chdir(old_path)
 
+class build(_build):
+    def run(self):
+        """
+        Run ./configure first and update libraries depending
+        on the generated output.
+        """
+        subprocess.check_call(["make", "configure"])
```

Comment:
I am a bit confused by this. `make configure` just generates the `configure` script, but does not run it. Is that intentional? Do you mean do execute `make dist` here, which executes `configure`?

Reply:
It is intentional (and standard). `make X` builds `X`. It does not run it.

Reply:
I can add a `RUN_configure` target. But it is better to separate the steps in the `setup.py` (more meaningful error messages).

Reply:
Yep, I understand. What I do not understand is the logic here.

You execute `make configure` which builds the configure script, but does not execute it. I thought you wanted to execute the configure script, which is not done in this `setup.py` step. So the `configure` never gets executed by `setup.py` (or at least it seems like it)

Reply:
Yes, it is the lines just after
```
        try:
            subprocess.check_call(["sh", "configure"])
        except subprocess.CalledProcessError:
            subprocess.check_call(["cat", "config.log"])
            raise
```

Reply:
Ah, okay, missed that, sorry :)